applications/protocols_serialization/client/src/ot_shell.c

```diff
@@ -59,6 +61,46 @@ static int ot_cli_command_invoke(ot_cli_command_handler_t handler, const struct
 	return 0;
 }
 
+/*
+ * Sends the command to the RCP server as a line of text. The command is then processed by
+ * the RPC server and its result is returned asynchronously using OT CLI output callback.
+ *
+ * This is a fallback function for commands that cannot be processed by the RPC client
+ * because serialization of the underlying OpenThread APIs is not yet implemented.
+ */
+static int ot_cli_command_send(const struct shell *sh, size_t argc, char *argv[])
+{
+	char buffer[256];
```

Comment:
the buffer size in general is fine for now however for dataset related commands it won't be sufficient. 

https://github.com/openthread/openthread/blob/main/examples/apps/cli/cli_uart.cpp#L69

Reply:
The maximum dataset size is 254, so most of them will fit :). And bigger than that won't be possible to input because of `CONFIG_SHELL_CMD_BUFF_SIZE` that is by default 256.

Reply:
Replaced 256 with CONFIG_SHELL_CMD_BUFF_SIZE to make sure this is never too small.